zenith_utils/src/pq_proto.rs

```diff
@@ -37,19 +37,21 @@ pub enum FeMessage {
 }
 
 #[derive(Debug)]
-pub struct FeStartupMessage {
-    pub version: u32,
-    pub kind: StartupRequestCode,
-    // optional params arriving in startup packet
-    pub params: HashMap<String, String>,
+pub enum FeInitialMessage {
+    CancelRequest(CancelKeyData),
+    SSLRequest,
+    GSSENCRequest,
```

Comment:
nit: I'd spell those as `SslRequest` and `GssEncRequest`, cause it's easier on the eyes. I know this is how they're spelled in the docs, but I'm not sure we're after one-to-one correspondence here.